package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@elastic/node-crypto",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
optional nit: since we're augmenting the API, should this be considered a minor release, rather than a patch?

Reply:
Yeah, fair point, done